app/controllers/state_file/intake_logins_controller.rb

```diff
@@ -32,8 +37,14 @@ def update
       end
     end
 
+    def account_locked; end
+
     private
 
+    def redirect_locked_clients
```

Comment:
commenting out this redirect for now b/c running into bug, will resolve in separate PR